packages/client/src/pages/Home/Home.tsx

```diff
@@ -334,6 +336,42 @@ export function Home({ engine }: IHome) {
                     />
                   </Box>
                 </Box>
+                <Typography
+                  variant="h4"
+                  sx={{
+                    padding: "60px 0 60px 128px",
+                    textAlign: "left",
+                  }}
+                >
+                  Несколько типов заклинаний:
+                </Typography>
+                <Box className="b-text-background">
+                  {Object.entries(engine.predefinedSpellParams).map((spell) => {
+                    const spellType = spell[0] as TSpellTypes;
+                    return (
+                      <Box
+                        sx={{ display: "flex" }}
+                        key={`b-spell-${spellType}-image-wrapper`}
```

Comment:
`key` - это внутренний атрибут, который используется движком React. Он не пробрасывается в компонент и не отображается в дом дереве. А это значит, что добавление строки в этом списке не имеет никакой ценности для движка
```suggestion
                        key={spellType}
```

Reply:
для движка - да, для того кто потом будет это читать - нет